tests/network/samba/samba_adcli.pm

```diff
@@ -125,6 +133,7 @@ sub run {
 
     samba_sssd_install();
     randomize_hostname();    # Prevent race condition with parallel test runs
+    disable_ipv6();    # AD host is not reachable via IPv6 on some of our workers
```

Comment:
I recall this was also done for s390 too... do we have a POO ticket to reference the issue? no need to soft-fail, but ipv6 should work too..

Reply:
Now we do :slightly_smiling_face:  https://progress.opensuse.org/issues/126482